recipe/meta.yaml

```diff
@@ -17,25 +17,30 @@ build:
 
 requirements:
   host:
-    - python >=2.7
+    - python >=3.7
     - pip
     - setuptools >=30.3.0
+    - setuptools_scm >=6
     - pyct >=0.4.4
     - wheel
 
   run:
-    - python >=2.7
+    - python >=3.7
     - pyct >=0.4.4
```

Comment:
`pyct` needs to be removed as it's no longer a runtime dependency of colorcet, see https://github.com/holoviz/colorcet/blob/9a7529835d376a866d62f3bc12552e695bbce751/pyproject.toml#L33